docs/docs/edsl.md

```diff
@@ -323,24 +320,27 @@ We determined the Tile code for this example by starting from imperative code,
 but this Tile code is still very similar to mathematical notation, and we could
 have started there instead:
 
-```math
+
+$$
 \Large
-\textcolor{red}{O[n]}
-\textcolor{yellow}{=}
-\textcolor{green}{\max}\textcolor{magenta}{_{0 \leq j < 2}}
-\textcolor{cyan}{I[2i + j]}
-```
+\color{red} O[n]
+\color{yellow} = 
+\color{green} \max_{\color{magenta} 0 \leq j < 2}
+\color{cyan} I[2i + j] 
+$$
+
 
-```math
+$$
 \Large
 \texttt{
-  if (\textcolor{magenta}{j < 2}) \{{
```

Comment:
This is why we need the `{% raw %}` block, to prevent Liquid from breaking on the double `{{`, which is needed to make open curly display properly in LaTeX.